guide/designing-products/user-research.md

```diff
@@ -84,21 +83,24 @@ There are many ways in which research and development are similar to other digit
    layout = "float-right-desktop"
 %}
 
-Most digital products start with a specific country, culture, language or legal framework in mind. This is especially true for financial applications, which are typically highly regulated and need to slowly and strategically evolve. While Bitcoin did start with a single language, its use case was not tied to any particular context, and this has resulted in a much more diverse user base from the start.
+When designers begin to develop a product, they often start with a specific country, culture, language, or legal framework in mind. This is especially true for financial applications, which are highly regulated and need to evolve slowly and strategically. While Bitcoin was originally developed for an English speaking audience, it didn’t take long for its intended use case to naturally proliferate it.
 
-This has resulted in diverse and lively online communities that are deeply invested in Bitcoin. Designers and researchers have vast amounts of public conversation at their hands that help them better understand user needs and issues. These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.
+Online communities rapidly formed and multiplied around Bitcoin. Vast amounts of public conversation helped designers and developers comprehend what was possible, and what was needed from Bitcoin.
```

Comment:
Not sure why this is written in the past now. These communities are still active and super helpful for user research today.

Reply:
Good point.

Reply:
@GBKS 

> Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.


The later paragraph though explains that these are still available, what do you suggest we do here?

Reply:
> These communities are also great for direct interaction to brainstorm ideas, get feedback on potential solutions, and get users involved in the design process.

I think we should keep the point being made here. Lots of designers don't realize that a great source of user feedback is so easily available to them. They look for formal research documents, when all they need to do is to tap into the online communities that are just a click away. It's something that I'd like to stress further in future discussions of user research.

Let me try to combine the brevity of the rewrite with the points I think we should make.

> Diverse online communities rapidly formed around Bitcoin. Vast amounts of public conversation can help designers and developers comprehend what is possible, and what is needed from Bitcoin. These communities are great for brainstorming ideas, getting feedback, and involving users actively in the design process.

And then I'd remove the next sentence, which is this one:

> Some of these links have been around for a while. The vast archives that they lead to still provide enormous value to anyone who wants to build Bitcoin products.

Reply:
@GBKS  Addressed in https://github.com/BitcoinDesign/Guide/pull/365/commits/66c0c6f3821190af411958971a2342857cfa0f3a